### DISCUSSION TITLE
Amiberry controls not working

### DISCUSSION THREAD

**Discussion by leomontenegro6** [wummle/arkos#6] [Q&A]
I was testing Amiga games through Amiberry. They run fine, but the controls aren't working no matter which buttons I press. Does anyone get a similar issue?

The games I tried are initially Lotus and Turrican games. I also tried going to Input and changing port 0 to "OpenSimHardware OSH PB Controller" and port 1 to "mouse", but the issue persists.

Would it be somehow related to different Kickstart ROM versions? The one I'm using is "Kickstart v1.3 rev 34.5 (1987)(Commodore)(A500-A1000-A2000-CDTV).rom", whose crc32 is "c4f0f55f". If true, which version(s) should be best suited?

**Comment by wummle** [comment]
Suggested to use PUAE instead Problem is lack of mouse emulation
